File: day2/day-1.js

function get_presentation_info(name,role)
{

	var name = name.trim();
	var role = role.trim();

	if(!name || !role){
		
		return 'Name and role are required';
	}

	return `Hi, My name is ${name} and I work as a ${role}`;

}

console.log(get_presentation_info('Ysidro Almonte','Web Developer'));

File: day1/day-1.py

def get_presentation_info(name,role):
	name = name.strip()
	role = role.strip()

	if not name or not role:
		return 'Name and role are required';

	return f'Hi, My name is {name.strip()} and I work as a {role.strip()}.'; 


print(get_presentation_info(' Ysidro Almonte ','Web Developer '))